example_9/test/test_view_robot_launch.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 FZI Forschungszentrum Informatik
```

Comment:
Dear @christophfroehlich thanks a lot for this very nice addition.
However I believe that all the licenses part is a copy pase that should be removed.
First it is a BSD Clause 2 license, while APACHE is the license claimed in the root license file.
Second the author is somebody else...

As this repository is for people learning ros2_control and that unit test is very important, removing this unnecessary part will help to keep the example clear.

Again thanks a lot for this welcome addition.

Reply:
Thanks for the hint, I'm not sure about the license stuff and how ament_copyright deals with it.. But I can try ;) You mean that I just remove the license part but let the line with the copyright claim?